easybuild/easyconfigs/a/AMDSMI/AMDSMI-6.4.1-LLVMtc-ROCm-6.4.1.eb

```
@@ -0,0 +1,39 @@
+easyblock = 'CMakeMake'
+
+name = 'AMDSMI'
```

```suggestion
name = 'amdsmi'
```

Use name of existing `amdsmi` EasyConfigs